src/prompts/quality_check.py

```
@@ -0,0 +1,157 @@
+            logger.error(error_msg)
+            raise ValueError(error_msg) from e
+        except Exception as e:
+            error_msg = f"品質チェック用プロンプトの読み込み中に予期せぬエラーが発生しました: {e}"
+            logger.error(error_msg)
+            raise RuntimeError(error_msg) from e
+    
+    
+    @classmethod
+    def get_comparative_quality_prompt(
+        cls,
+        original_text: str,
```

Ruff (0.14.13)

Unused class method argument: `original_text`

(ARG003)